src/main/java/Orca/Storage.java

```diff
@@ -0,0 +1,133 @@
+package Orca;
+
+import Orca.tasktype.Deadline;
+import Orca.tasktype.Event;
+import Orca.tasktype.Todo;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    // Use OS-independent file separators.
```

Comment:
These comments seem to be for your reference instead of the reader. You may want to erase them